LiveSplit.AutoSplitters.xml

```diff
@@ -14654,9 +14654,12 @@
         </Games>
         <URLs>
             <URL>https://raw.githubusercontent.com/PoPRuns/PoP.AutoSplitters/main/multipop.asl</URL>
+            <URL>https://raw.githubusercontent.com/PoPRuns/PoP.AutoSplitters/main/pop_sot.asl</URL>
+            <URL>https://raw.githubusercontent.com/PoPRuns/PoP.AutoSplitters/main/pop_ww.asl</URL>
+            <URL>https://raw.githubusercontent.com/PoPRuns/PoP.AutoSplitters/main/pop_t2t.asl</URL>
```

Comment:
This change doesn't make sense. Why are you downloading multiple asl files for one autosplitter?

Reply:
I strongly disagree with that approach.

Reply:
Is it purely for the settings?

Reply:
Well, it avoids having to maintain splits in two places. And for such a rarely touch category extension, it's good enough.

Reply:
You could offload the settings to an entirely separate file which is loaded by both the individual ASL and the multi-ASL.  
I see a flaw in your approach in that when the runner changes any settings in the multi-ASL while one of the games is running, this change is not translated over to the ASL on the layout.

Reply:
I could move the settings to another file, but then the splits logic would still have to be written in multipop.asl again. And updated if it's updated in the individual games.

Yes I am aware of this limitation. The solution is to set settings before starting a run.

Reply:
I could move the settings to another file, but then the splits logic would still have to be written in multipop.asl again. And updated if it's updated in the individual games.

Yes I am aware of this limitation. The solution is to set settings before starting a run, I have made the runners aware of it too.

Reply:
You are also removing the ASL component when the timer resets. I'm not sure that's appropriate.

Reply:
Feel free to try again another time. I think the idea is fine, the implementation just needs some work.